make/agents/svc/deploy.mk

```diff
@@ -0,0 +1,14 @@
+KUSTOMIZE_ARGS=--load-restrictor LoadRestrictionsNone
+KUSTOMIZE_OVERLAY=config/crd/overlays/agentsvc
+
+ifndef ignore-not-found
+  ignore-not-found = false
+endif
+
+.PHONY: install
+install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
+	$(KUSTOMIZE) build $(KUSTOMIZE_ARGS) $(KUSTOMIZE_OVERLAY) | kubectl apply -f -
+
+.PHONY: uninstall
+uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
+	$(KUSTOMIZE) build $(KUSTOMIZE_ARGS) $(KUSTOMIZE_OVERLAY) | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
```

Comment:
Q: how is a developer intended to deploy an agent for testing purposes?  I don't see a deploy rule defined anywhere.

Reply:
nice catch, added!